src/reference/01-General-Info/07-Contributing-to-sbt.md

```diff
@@ -1,5 +1,5 @@
 ---
-out: Contibuting-to-sbt.html
```

Comment:
Not sure about this one, since it will eventually break some links in the wild wild web.

Reply:
I think it's ok.